### PR TITLE
Remove `./third_party/config/build/release.yaml` from DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by zhyuri**
As suggest in the https://github.com/knative/build-pipeline/issues/390, remove `./third_party/config/build/release.yaml` related lines from DEVELOPMENT.md.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -140,7 +140,6 @@ by [`Task`](docs/Concepts.md#task)):
 
 ```shell
 ko apply -f config/
```
Comment (pivotal-nader-ziada) [inline DEVELOPMENT.md]: can we also remove the description above that line that says `, including knative/build (which is wrapped by Task):`

Comment (zhyuri) [reply DEVELOPMENT.md]: Sure.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -135,12 +135,10 @@ To make changes to these CRDs, you will probably interact with:
 ## Install Pipeline
 
 You can stand up a version of this controller on-cluster (to your
-`kubectl config current-context`), including `knative/build` (which is wrapped
-by [`Task`](docs/Concepts.md#task)):
+`kubectl config current-context`)):
```
Comment (pivotal-nader-ziada) [inline DEVELOPMENT.md]: sorry for the nit, but it has two closing brackets `))`

Comment (zhyuri) [reply DEVELOPMENT.md]: Oh shoot, I'm so sorry for that. 
Thanks a lot for pointing out.

**Comment by pivotal-nader-ziada** [comment]
Thanks for the changes @zhyuri 
/lgtm
/approve

**Comment by bobcatfish** [comment]
Thanks @zhyuri !! 🙏

**Pull request merged**